sw/device/tests/sysrst_ctrl_ulp_z3_wakeup_test.c

```diff
@@ -51,30 +58,44 @@ static const dif_pinmux_index_t kPeripheralInputs[] = {
     kTopEarlgreyPinmuxPeripheralInSysrstCtrlAonLidOpen,
 };
 
-static const dif_pinmux_index_t kInputPads[] = {
+static const dif_pinmux_index_t kInputPadsDV[] = {
     kTopEarlgreyPinmuxInselIor13,
     kTopEarlgreyPinmuxInselIoc7,
     kTopEarlgreyPinmuxInselIoc9,
 };
 
+static const dif_pinmux_index_t kInputPadsReal[] = {
+    kTopEarlgreyPinmuxInselIor10,
+    kTopEarlgreyPinmuxInselIor11,
+    kTopEarlgreyPinmuxInselIor12,
+};
+
 static const dif_pinmux_index_t kPeripheralOutputs[] = {
     kTopEarlgreyPinmuxOutselSysrstCtrlAonZ3Wakeup,
 };
 
-static const dif_pinmux_index_t kOutputPads[] = {
+static const dif_pinmux_index_t kOutputPadsDV[] = {
     kTopEarlgreyPinmuxMioOutIob7,
 };
 
+static const dif_pinmux_index_t kOutputPadsReal[] = {
+    kTopEarlgreyPinmuxMioOutIor5,
+};
+
 /**
  * Sets up the pinmux to assign input and output pads to the sysrst_ctrl
  * peripheral as required.
  */
 static void pinmux_setup(void) {
+  const dif_pinmux_index_t *kInputPads =
```

Comment:
I'm not sure, is `kInputPads` a compile time const? If not it should use snake_case .

Reply:
It is in the sense that `kDeviceType` is a constant variable defined in the arch header. Unfortunately, it is not technically a "constant expression" in the C sense because `kDeviceSimDV` is a const variable and not a macro, therefore `kInputPads` cannot be defined at the global scope.